res/skins/Deere/sampler_controls_row.xml

```diff
@@ -85,7 +85,7 @@
           </Template>
 
           <Template src="skin:left_right_1state_button.xml">
-            <SetVariable name="TooltipId">beatsync_beatsync_tempo</SetVariable>
+            <SetVariable name="TooltipId">sync_enabled</SetVariable>
```

Comment:
You are only changing the tooltip here. To replace the functionality of the button, you also have to replace `left_connection` below.
```xml
            <SetVariable name="left_connection_control"><Variable name="group"/>,sync_enabled</SetVariable>
```
 Note that there is corresponding replacement for `beatsync_tempo`, so the `right_connection` should be removed. See issue comments for further discussion.